pkg/csi/cinder/controllerserver.go

```diff
@@ -257,19 +259,24 @@ func (d *controllerServer) ControllerModifyVolume(ctx context.Context, req *csi.
 func (cs *controllerServer) DeleteVolume(ctx context.Context, req *csi.DeleteVolumeRequest) (*csi.DeleteVolumeResponse, error) {
 	klog.V(4).Infof("DeleteVolume: called with args %+v", protosanitizer.StripSecrets(req))
 
+	// Get VolumeID to be deleted
+	volID := req.GetVolumeId()
+	if len(volID) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "DeleteVolume Volume ID must be provided")
+	}
+
```

Comment:
Again, there's no obvious reason to move this. If you think it should be moved, please do so in a separate commit with an explanation for why. Otherwise, please revert this.